Dockerfile.template

```diff
@@ -149,6 +149,7 @@ RUN set -eux; \
 		| sort -u \
 		| xargs -r apt-mark manual \
 	; \
+	apt-mark manual libusbredirparser1; \
```

Comment:
This is definitely odd, and I think we need to adjust our loop above instead to also catch `.so` files because this is definitely a "missing dependency" that should've been captured :disappointed::

```console
root@e7f3b33668cf:/# ldd /usr/local/lib/qemu/hw-usb-redirect.so | grep 'not found'
	libusbredirparser.so.1 => not found
```

Changing `find /usr/local -type f -executable -exec ldd ...` to `find /usr/local -type f \( -executable -o -name '*.so' \) -exec ldd ...` properly pulls in `libusbredirparser1` and changes the 5.1 image size from ~531MB to ~534MB (which is still reasonable, but probably screws up https://github.com/tianon/docker-qemu/pull/11#issuecomment-689816553 :weary:).

Reply:
Confirmed, #11 makes this more complicated (and results in a ~599MB image :grimacing:), so I think this also needs to account for that, probably something like:

```diff
diff --git a/Dockerfile.template b/Dockerfile.template
index 60e2ac7..b3de387 100644
--- a/Dockerfile.template
+++ b/Dockerfile.template
@@ -118,6 +118,7 @@ RUN set -eux; \
 		--enable-linux-user \
 		--enable-modules \
 		--enable-numa \
+		--enable-rbd \
 		--enable-seccomp \
 		--enable-system \
 		--enable-tools \
@@ -131,8 +132,6 @@ RUN set -eux; \
 		--enable-vnc-png \
 		--enable-xen \
 		--enable-xfsctl \
-# rbd support is enabled, but "librbd1" is not included since it adds ~60MB and is version-sensitive (https://github.com/tianon/docker-qemu/pull/11#issuecomment-689816553)
-		--enable-rbd \
 #		--enable-vde \
 	; \
 	make -j "$(nproc)"; \
@@ -143,7 +142,12 @@ RUN set -eux; \
 	\
 	apt-mark auto '.*' > /dev/null; \
 	[ -z "$savedAptMark" ] || apt-mark manual $savedAptMark > /dev/null; \
-	find /usr/local -type f -executable -exec ldd '{}' ';' \
+	find /usr/local \
+		-type f \
+		\( -executable -o -name '*.so' \) \
+# rbd support is enabled, but "librbd1" is not included since it adds ~60MB and is version-sensitive (https://github.com/tianon/docker-qemu/pull/11#issuecomment-689816553)
+		-not -name 'block-rbd.so' \
+		-exec ldd '{}' ';' \
 		| awk '/=>/ { print $(NF-1) }' \
 		| sort -u \
 		| xargs -r dpkg-query --search \
```

Which results in 5.1 being back down to ~534MB.

Reply:
I've added the changes and updated the commit message. I'm not really happy about the exclusion, but I realize that the image size is of importance.